frontend/src/reviews/EditableReviewComment.tsx

```diff
@@ -0,0 +1,53 @@
+import { Button, Stack, Typography } from '@mui/material'
+import MarkdownDisplay from 'src/common/MarkdownDisplay'
+import RichTextEditor from 'src/common/RichTextEditor'
+import MessageAlert from 'src/MessageAlert'
+import { ResponseInterface } from 'types/types'
+import { formatDateTimeString } from 'utils/dateUtils'
+
+interface EditableReviewCommentProps {
+  comment: string
+  setComment: (value: string) => void
+  response: ResponseInterface
+  isEditMode: boolean
+  setIsEditMode: (value: boolean) => void
+  editCommentErrorMessage: string
+  onSave: () => void
+}
+
+export default function EditableReviewComment({
+  comment,
+  setComment,
+  response,
+  isEditMode,
+  setIsEditMode,
+  editCommentErrorMessage,
+  onSave,
+}: EditableReviewCommentProps) {
+  return (
+    <>
+      {comment && !isEditMode && (
```

Comment:
`comment` is defined as a required string, so the check `comment &&` is unnecessary.